admin/templates/users/user.html

```diff
@@ -233,6 +233,17 @@ <h3>User details</h3>
                         {% for system_tag in user.system_tags %}
                             {{ system_tag }}{% if not forloop.last %}, {% endif %}
                         {% endfor %}
+                    <a href="{% url 'users:add_system_tag' user.id %}"
```

Comment:
Here's what the button looks like:
<img width="420" alt="screen shot 2019-03-07 at 10 42 46 am" src="https://user-images.githubusercontent.com/801594/53968890-d81c9e80-40c5-11e9-9cc6-450695a78a17.png">

And the modal:
<img width="608" alt="screen shot 2019-03-07 at 10 43 37 am" src="https://user-images.githubusercontent.com/801594/53968923-e4a0f700-40c5-11e9-86f8-319a2f041d37.png">

Reply:
Is there a way to remove system tags from users?

Reply:
Not at the moment, nope, except manually of course. Thought about it, but basically didn't have time to fit it in here.